x-pack/platform/plugins/shared/fleet/server/services/package_policies/package_policies_aggregation.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectsClientContract } from '@kbn/core/server';
+
+import { getPackagePolicySavedObjectType } from '../package_policy';
+
+export async function getPackagePoliciesCountByPackageName(soClient: SavedObjectsClientContract) {
+  const savedObjectType = await getPackagePolicySavedObjectType();
+
+  const res = await soClient.find<
+    any,
+    { count_by_package_name: { buckets: Array<{ key: string; doc_count: number }> } }
+  >({
+    type: savedObjectType,
+    perPage: 0,
```

Comment:
should we add `size:0` since `hits` are not used?

Reply:
yes 👍  I will make the change

Reply:
actually it's already done with the `perPage: 0` it's a saved object client not an ES client